content/en/blog/2023/istiocon-china/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "IstioCon China 2023 wrap-up"
+description: A quick recap of Istio at KubeCon + CloudNativeCon + Open Source Summit China, in Shanghai.
+publishdate: 2023-09-29
+attribution: "IstioCon China 2023 Programme Committee"
```

Comment:
You have used both programme and program for the committee in this doc. I'm not sure what the official title is?

Reply:
Fixed it